---
int16 encoder_value